test_scripts/Defects/7_1/1600_SetGlobalProperties_UNSUPPORTED_RESOURCE_response_success.lua

```diff
@@ -0,0 +1,71 @@
+---------------------------------------------------------------------------------------------------
+-- Issue: https://github.com/SmartDeviceLink/sdl_core/issues/1585
+---------------------------------------------------------------------------------------------------
+-- Description: HMI responds with UNSUPPORTED_RESOURCE resultCode to SetGlobalProperties component
+--
+-- Preconditions:
+-- 1) Clean environment
+-- 2) SDL, HMI, Mobile session are started
+-- 3) App is registered
+-- 4) App is activated
+--
+-- Steps:
+-- 1) Send SetGlobalProperties mobile RPC from app
+-- 2) HMI responds with UNSUPPORTED_RESOURCE to TTS.SetGlobalProperties request
+--
+-- SDL does:
+-- 1) Respond with 'SetGlobalProperties' (success = true, resultCode = "UNSUPPORTED_RESOURCE") to App
+---------------------------------------------------------------------------------------------------
+
+--[[ Required Shared libraries ]]
+local runner = require('user_modules/script_runner')
+local common = require("user_modules/sequences/actions")
+
+--[[ Test Configuration ]]
+runner.testSettings.isSelfIncluded = false
+
+--[[ Local Variables ]]
+local params = {
+  helpPrompt = {
+    {
+      type = "SAPI_PHONEMES",
+      text = "Text to Speak"
+    }
+  },
+  menuTitle = "Hello, driver!"
+}
+
+--[[ Local Functions ]]
+local function sendSetGlobalProperties()
+  local mobileSession = common.mobile.getSession()
+  local hmi = common.hmi.getConnection()
+  local dataToHMI = {
+    appID = common.app.getHMIId()
+  }
+
+  local cid = mobileSession:SendRPC("SetGlobalProperties", params)
+  hmi:ExpectRequest("UI.SetGlobalProperties", dataToHMI)
```

Comment:
Checking of just `appID` in `dataToHMI` is useless. It would be better to exclude verification on data in request completely (in the same way as for `TTS.SetGlobalProperties`).